proctor-consumer/src/main/java/com/indeed/proctor/consumer/ProctorConsumerUtils.java

```diff
@@ -76,14 +79,14 @@ public static ProctorResult determineBuckets(
      * @return a map of test names to bucket values specified by the request.  Returns an empty {@link Map} if nothing was specified
      */
     @Nonnull
-    public static ForceGroupsOptions parseForcedGroupsOptions(@Nonnull final HttpServletRequest request) {
+    public static ForceGroupsOptions parseForcedGroupsOptions(@Nonnull final HttpServletRequest request, final Set<String> forcePayloadTests) {
         final String forceGroupsList = getForceGroupsStringFromRequest(request);
-        return ForceGroupsOptionsStrings.parseForceGroupsString(forceGroupsList);
+        return ForceGroupsOptionsStrings.parseForceGroupsString(forceGroupsList, forcePayloadTests);
     }
 
     @Nonnull
-    public static Map<String, Integer> parseForceGroupsList(@Nullable final String payload) {
-        return ForceGroupsOptionsStrings.parseForceGroupsString(payload)
+    public static Map<String, Integer> parseForceGroupsList(@Nullable final String payload, final Set<String> forcePayloadTests) {
```

Comment:
As same as the above comment, users of this method will get a compile error.

In this case, I suggest keeping `public static Map<String, Integer> parseForceGroupsList(@Nullable final String payload)` and don't add the new signature until we find it's necessary to add.